src/cryptonote_core/service_node_list.cpp

```diff
@@ -952,6 +980,15 @@ namespace service_nodes
       info.total_reserved += resultlo;
     }
 
+    // In HF16 we require that the amount staked in the registration tx be at least the amount
+    // reserved for the operator.  Before HF16 it only had to be >= 25%, even if the operator
+    // reserved amount was higher (though wallets would never actually do this).
```

Comment:
Not that wallets do it in one go but this was to allow someone to incrementally stake the rest if they went and attained some more. Not sure how much in practice this was used. We may also be able to apply this without the gating after the hardfork.

Incremental staking is not needed anymore with stake unlocking as you just go and unlock and redo your registration.

Reply:
They still can increase later, but the stake that increases it has to follow all the usual requirements for any stake output (i.e. must be 1/N of the remaining amount).  This was just to cover a registration tx that says it reserves 10k for the operator but only actually stakes 8k in that tx -- given the current rules it seemed like that could result in an impossible registration because that extra stake to top off the registration could end up being rejected as a fifth stake output.